src/main/java/dook/commands/TaskType.java

```diff
@@ -0,0 +1,5 @@
+package dook.commands;
+
+public enum TaskType {
+    TODO, EVENT, DEADLINE
+};
```

Comment:
Should this class be moved to the task package since the enum TaskType is related to tasks?